Maple2.Server.Game/Manager/AchievementManager.cs

```diff
@@ -244,4 +244,20 @@ public void Save(GameStorage.Request db) {
         db.SaveAchievements(session.AccountId, accountValues.Values.ToList());
         db.SaveAchievements(session.CharacterId, characterValues.Values.ToList());
     }
+
+    public void SaveNewAchievement(Achievement achievement, bool isAccountWide) {
```

Comment:
This function scares me a bit because it can cause confusion. I don't want this to be something people think is how achievements are saved, nor is there anywhere else that would use this function other than a gm command. You should just move all this code into the PlayerCommand class